eclair-node-gui/src/main/resources/gui/main/main.fxml

```diff
@@ -88,15 +88,21 @@
                                                              minWidth="120.0" prefWidth="170.0" maxWidth="200.0"
                                                              text="Short Channel Id"/>
                                                 <TableColumn fx:id="networkChannelsNode1Column" text="Node 1"/>
+                                                <TableColumn fx:id="networkChannelsFeeBaseMsatNode1Column"
+                                                             minWidth="100.0" prefWidth="120.0" maxWidth="200.0"
+                                                             text="Base Fee"/>
+                                                <TableColumn fx:id="networkChannelsFeeProportionalMillionthsNode1Column"
+                                                             minWidth="60.0" prefWidth="120.0" maxWidth="200.0"
+                                                             text="Proportional Fee"/>
                                                 <TableColumn fx:id="networkChannelsDirectionsColumn"
                                                              minWidth="30.0" prefWidth="30.0" maxWidth="30.0"/>
                                                 <TableColumn fx:id="networkChannelsNode2Column" text="Node 2"/>
                                                 <TableColumn fx:id="networkChannelsCapacityColumn" text="Capacity"
                                                              minWidth="80.0" prefWidth="120.0" maxWidth="200.0"/>
-                                                <TableColumn fx:id="networkChannelsFeeBaseMsatColumn"
+                                                <TableColumn fx:id="networkChannelsFeeBaseMsatNode2Column"
                                                              minWidth="100.0" prefWidth="120.0" maxWidth="200.0"
                                                              text="Base Fee"/>
-                                                <TableColumn fx:id="networkChannelsFeeProportionalMillionthsColumn"
+                                                <TableColumn fx:id="networkChannelsFeeProportionalMillionthsNode2Column"
                                                              minWidth="60.0" prefWidth="120.0" maxWidth="200.0"
                                                              text="Proportional Fee"/>
                                             </columns>
```

Comment:
* `text` field in new columns should be `Node 1 Base Fee`, `Node 1 Proportional Fee`, `Node 2 Base Fee` and `Node 2 Proportional Fee`
* `networkChannelsCapacityColumn` column should be moved at the end, or at the beginning, after the short channel id column